### PR TITLE
Deleting old cluster once the migration is complete

### PR DESCRIPTION
**Pull request by rchikatw**

### CODE REVIEW

File: migrate.sh [hunk 1]
```diff
@@ -116,4 +116,17 @@ done
...
 sh ./updateEBSVolumeTags.sh
 
+echo -e "\nDeleting the old/backup cluster"
+loginCluster $1 "$backupClusterID"
+consumers=( $(kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}') )
+for consumer in $consumers
+do
+  kubectl patch storageconsumers  ${consumer} -n openshift-storage -p '{"metadata":{"finalizers":null}}' --type=merge
+  kubectl delete storageconsumers  ${consumer} -n openshift-storage
+done
```
Comment (rewantsoni) [inline migrate.sh]: We can replace this with
```suggestion
kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl patch storageconsumer -p '{"metadata":{"finalizers":null}}' --type=merge -n openshift-storage
kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl delete storageconsumer -n openshift-storage
```

File: migrate.sh [hunk 2]
```diff
@@ -116,4 +116,17 @@ done
...
+clusterName=$(ocm list clusters | grep ${backupClusterID} | awk '{print $2}')
+serviceId=$(rosa list services | grep resoni-p1 | awk '{print $1}')
```
Comment (rewantsoni) [inline migrate.sh]:
```suggestion
serviceId=$(rosa list services | grep ${clusterName} | awk '{print $1}')
```

**Review by rewantsoni — CHANGES_REQUESTED** [review]
We should add rosa to the list of commands we require

**Comment by rewantsoni** [comment]
@rchikatw Did you verify the the changes for un installation? Does only patching storageconsumer works?

File: migrate.sh (hunk 1 above)
Comment (rewantsoni) [reply migrate.sh]: We also need to patch storageSystem and storagecluster

Comment (rchikatw) [reply migrate.sh]: Sure

**Pull request merged**